### ISSUE TITLE
Resolving of relative links of the demo fails for vaadin.com

### ISSUE THREAD

**Issue by paulroemer** [vaadin/vaadin-tabs#75]
During bundling the relative URLs defined in `my-custom-icons.html`
```
<link rel="import" href="../../polymer/polymer-element.html">
<link rel="import" href="../../vaadin-themable-mixin/vaadin-themable-mixin.html">
```
are resolved to `src/main/webapp/frontend/polymer/polymer-element.html`. The `bower_components` path fragment is missing.

**Comment by paulroemer** [comment]
The solution is to change the import of `my-custom-icons.html` in index.html:

```
<link rel="import" href="../demo/my-custom-item.html">
```
instead of
```
<link rel="import" href="my-custom-item.html">
```

**Comment by web-padawan** [comment]
Fixed by #76
